Pages/Index.cshtml.cs

```diff
@@ -13,40 +13,69 @@ namespace NewYork_CIty_School_Data_With_Crime_Rate_History.Pages
 {
     public class IndexModel : PageModel
     {
+        public string GetData(string endpoint)
+        {
+            string downloadedData = "";
+            using (WebClient webClient = new WebClient())
+            {
+                downloadedData = webClient.DownloadString(endpoint);
+            }
+            return downloadedData;
+        }
         public JsonResult OnGet()
         {
 
             List<MergedData> mergedData = new List<MergedData>();
 
             using (WebClient webClient = new WebClient())
             {
-                string schoolJsonString = webClient.DownloadString("https://data.cityofnewyork.us/resource/23z9-6uk9.json");
+                string schoolJsonString = GetData("https://data.cityofnewyork.us/resource/23z9-6uk9.json");
+                //               string schoolJsonString = webClient.DownloadString("https://data.cityofnewyork.us/resource/23z9-6uk9.json");
                 SchoolData[] schoolData = SchoolData.FromJson(schoolJsonString);
 
-                string crimeJsonString = webClient.DownloadString("https://data.cityofnewyork.us/resource/kwvk-z7i9.json");
+                string crimeJsonString = GetData("https://data.cityofnewyork.us/resource/kwvk-z7i9.json");
+                //                string crimeJsonString = webClient.DownloadString("https://data.cityofnewyork.us/resource/kwvk-z7i9.json");
```

Comment:
I was thinking you could modify some of the naming used for classes. Drop the 'Data' and just call it School and Crime might be more cleaner. Then you can call the instance as 'schools' and 'crimes'. I thought that will be very close to real world

Reply:
You're in your own branch, so show how to do this in code... better than describing it in words.